src/main/java/domain/LottoPrize.java

```diff
@@ -0,0 +1,31 @@
+package domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LottoPrize {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
```

Comment:
```suggestion
    FIVE(5, 1_500_000),
```
언더스코어를 활용하여 가독성을 좀 더 확보할 수도 있습니다!

Reply:
오오 꿀팁 알려주셔서 갑사합니다 😄